content/applications/sales/crm/track_leads/lead_distribution_report.rst

```diff
@@ -0,0 +1,205 @@
+:guilabel:`Add Custom Filter`. This opens an :guilabel:`Add Custom Filter` pop-up window, where the
+essential filters can be added one at a time.
+
+.. _crm/track_leads/essential-filters:
+
+Essential filters
+-----------------
+
+The following filter conditions are used to create a basic lead distribution report. Together they
+gather all leads created within a certain timespan that have an associated contact method and have
+been assigned to a sales team.
+
+Lead creation date
+~~~~~~~~~~~~~~~~~~
+
+Click the first field, under :guilabel:`Match any of the following rules:`, that has the value
+:guilabel:`Country` in it. In the popover that appears, type `Created on` in the search bar, or
+scroll to search through the list to locate and select it.
+
+Then, in the second field of that row, select :guilabel:`>=` from the drop-down menu. This operator
+**only** includes values greater than (or equal to) the value in the third, rightmost field.
+
+The third field on the :guilabel:`Add Custom Filter` pop-up window should contain the earliest date
+leads are selected from.
+
+For example, setting `01/01/2024 00:00:00` only includes leads created from, and including, the
+first day of 2024.
+
+.. image:: lead_distribution_report/created-on.png
+  :align: center
+  :alt: Add a Created On rule for the start of the year onward.
+
+.. _crm/track_leads/sales-team:
+
+Sales team
+~~~~~~~~~~
+
+Click :guilabel:`New rule` to add another row to the form, and choose :guilabel:`Sales Team` for
+this rule's parameter. Then, click the second field of the new rule, and select :guilabel:`contains`
+from the drop-down menu. Selecting this operator filters for any records that contain the words in
+the third, rightmost field.
+
+.. tip::
+   For certain pre-determined, limited choices like a sales team, the :guilabel:`is in` operator
+   helps make for an easier and more accurate selection, via a drop-down menu in the third field,
+   instead of risking a typo or incorrect value in the text box field that accompanies the
+   :guilabel:`contains` operator.
+
+In this third field, enter the name of the desired sales team(s) that are to be included in the
+report. It is important for all :guilabel:`contains` argument values to be specific enough and
+spelled correctly as they exist in Odoo, otherwise this risks returning multiple (or zero) values.
+
+.. image:: lead_distribution_report/sales-team-location.png
+  :align: center
+  :alt: Use Sales Team to filter the location the lead is associated with.
+
+.. important::
+   By adding more than one rule to the form, a new option emerges at the top of the pop-up window
+   above all the filters, to specify whether :guilabel:`any` :icon:`fa-caret-down` :guilabel:`(down
+   caret)` or :guilabel:`all` :icon:`fa-caret-down` :guilabel:`(down caret)` of the conditions
+   should match. This distinction is important to set correctly, as it impacts the driving logic of
+   how the filters return data.
```

Comment:
In relation to @StraubCreative's [comment](https://github.com/odoo/documentation/pull/9915/files#r1696099002), there is no need to accompany an icon with it's descriptor/fallback if the UI element is already accompanied by text.

In this case, `any 🔽` and `all 🔽` have the text "any" or "all". So no need for the `(down caret)` descriptor :)
```suggestion
   above all the filters, to specify whether :guilabel:`any` :icon:`fa-caret-down` or
   :guilabel:`all` :icon:`fa-caret-down` of the conditions should match. This distinction is
   important to set correctly, as it impacts the driving logic of how the filters return data.
```